### DISCUSSION TITLE
chapter19 section79 の  /service/interface.go　について

### DISCUSSION THREAD

**Discussion by FUJI0130** [budougumi0617/go_todo_app#85] [Ideas]
書籍を見ながらハンズオンを進めていましたが、書籍上でこちらのファイルについて解説されておらず、
書籍に書いてる通りに進めると、この/service/interface.goの中のinterfaceを定義していないために先に進められないという状況になりました　幸い、githubリポジトリで確認や検索などさせて戴けたおかげで気づけましたが、書籍には書いていないという事をどこかで知らせる必要があるのではないかと思いまして投稿しました　私の勘違いでどちらかにこの件について書かれているようでしたら申し訳ありません

**Comment by FUJI0130** [comment]
すみません、
https://github.com/budougumi0617/go_todo_app/blob/main/errata.md
P216 「service」パッケージの実装
という所に書いてありました

**Comment by budougumi0617** [comment]
@FUJI0130 回答が遅くなってしまい申し訳ありません。ご確認いただいたとおり、他の方にもご指摘いただいておりました 🙇 
他にもわかりにくいところありましたら投稿していただければ回答いたします、本が少しでもお役に立てていれば幸いです！
